cscore/src/main/java/edu/wpi/first/cscore/CvSink.java

```diff
@@ -32,27 +32,12 @@ public void close() {
   }
 
   private int getCVFormat(PixelFormat pixelFormat) {
-    int type = 0;
-    switch (pixelFormat) {
-      case kYUYV:
-      case kRGB565:
-      case kY16:
-      case kUYVY:
-        type = CvType.CV_8UC2;
-        break;
-      case kBGR:
-        type = CvType.CV_8UC3;
-        break;
-      case kBGRA:
-        type = CvType.CV_8UC4;
-        break;
-      case kGray:
-      case kMJPEG:
```

Comment:
I'm not a fan of these two being inlined implicitly into the default case. They should be included explicitly as in the old impl.